root_profiler/engine/db.php

```diff
@@ -24,11 +24,7 @@ class Database
 	//
 	public function __construct($dsn, $username, $password)
 	{
-		$this->username = $username;
-		$this->password = $password;
-		$this->dsn = $dsn;
-
-		$this->dbh = new PDO($this->dsn, $this->username, $this->password);
+		$this->dbh = new PDO($dsn, $username, $password);
```

Comment:
So this no longer stores the username, password and whatever dsn is.
Ah, I see - in the lines above, this is a class being declared and those aren't declared as member variables.
Nor are they referenced anywhere else.